docs/source/tutorial.rst

```diff
@@ -33,8 +33,7 @@ container image for deployment.
 Setup for the tutorial
 ----------------------
 
-There are three ways to complete this tutorial: you can either run the code in browser
-with Google Colab, or you can set up a local development environment on your computer.
+There are three ways to complete this tutorial:
```

Comment:
request: should we make a table and hyperlink each section to the section below?

Reply:
the first two parts are very short that fits in one page, probably not worth adding another hyperlink section here